en/getting_started/ros.md

```diff
@@ -0,0 +1,12 @@
+# Generate Source Files for ROS
+
+If you want to add MAVlink messages or dialects while working with ROS, you need to:
+
+- follow the [instructions](https://github.com/mavlink/mavros/blob/master/mavros/README.md#source-installation) to install the mavlink-gbp-release which is the MAVlink library released for ROS.
+- if you have previously installed the MAVlink package for ROS, disinstall it
+```
+sudo apt-get remove ros-*-mavlink
```

Comment:
Quick and dirty alternative to this: `sudo apt-get remove ros-${rosversion -d}-mavlink` or `sudo apt-get remove ros-${ROS_DISTRO}-mavlink`. Though, you don't actually need to remove it. If you source your `devel/setup.bash` of your catkin workspace, it overrides the lib dir.